.github/workflows/size-impact/report-size-impact.js

```diff
@@ -1,9 +1,32 @@
-import { reportSizeImpactIntoGithubPullRequest } from "@jsenv/github-pull-request-filesize-impact";
+import {
+  reportFileSizeImpact,
+  readGithubWorkflowEnv,
+  none,
+  gzip,
+  brotli,
+} from "@jsenv/file-size-impact";
 
-reportSizeImpactIntoGithubPullRequest({
-  projectDirectoryUrl: new URL("../../../", import.meta.url),
-  baseSnapshotFileRelativeUrl: process.argv[2],
-  headSnapshotFileRelativeUrl: process.argv[3],
-  commentSections: { fileByFileImpact: true },
-  generatedByLink: true,
+reportFileSizeImpact({
+  ...readGithubWorkflowEnv(),
+
+  trackingConfig: {
+    browser: {
```

Comment:
Currently comment has 2 sections `core` and `extras`.

![image](https://user-images.githubusercontent.com/443639/90341212-f5190900-dffd-11ea-8e66-244ff354834c.png)

I took the liberty to split `core` into `browser` and `node`. 


With this changes there would be 3 sections in the comment: `browser`, `node` and `extras`.
This is because size impact on `system-node.cjs` are less important than ones on `s.min.js` or `system.min.js`.